packages/react/src/primitives/Select/Select.tsx

```diff
@@ -57,6 +59,8 @@ const SelectPrimitive: Primitive<SelectProps, 'select'> = (
             : defaultValue
         }
         isDisabled={isFieldsetDisabled ? isFieldsetDisabled : isDisabled}
+        multiple={isMultiple}
+        size={selectSize}
```

Comment:
Could you please specify what `view` file are you referring to? When I hover over that `size` prop I see that it expects a number, but may be I missed something.
<img width="404" alt="Screenshot 2023-09-11 at 3 22 32 PM" src="https://github.com/aws-amplify/amplify-ui/assets/19784835/78fceb4d-e49e-4cea-a367-7a666ec13f20">

Reply:
Oh cool, was under the impression that all of our `size` properties mapped to `Sizes`

Reply:
FYI the `size` that we're passing here is an HTML attribute for `<select>`. For example:

```
<View size="small">I'm a div</View>
```

results in a type error because size doesn't exist on HTMLDivElement. But

```
<View as="select" size={2}>I'm a select</View>
```

works because size exists as a type on HTMLSelectElement.

Not to be confused with the `size` prop that exists on our *Field primitives which maps to Sizes ('small' | 'large') which unfortunately has the same name. That is why we're using `selectSize` as the name of the prop that our SelectField accepts to set this attribute.

Reply:
Part of me feels like we are making usage DX confusing by using HTML attr names in a non-semantic way but discussed with @dbanksdesign and feel okay abt `selectSize`